crates/apub/src/activities/receive/private_message.rs

```diff
@@ -50,12 +50,19 @@ pub(crate) async fn receive_create_private_message(
     private_message_view: message,
   };
 
+  // Send notifications to the local recipient, if one exists
   let recipient_id = res.private_message_view.recipient.id;
+  let local_recipient_id = blocking(context.pool(), move |conn| {
+    LocalUserView::read_person(conn, recipient_id)
+  })
+  .await??
+  .local_user
+  .id;
```

Comment:
It can't just forward that `private_message_view.recipient.id`, (which is a `person_id`), it needs to do a read to find the local user, to get the `local_user_id`,

Reply:
You might also move this and the do_send below into a helper method.